snippets/ztp-the-policygenerator-single.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+    name: group-du-sno
+placementBindingDefaults:
+    name: group-du-sno-placement-binding
+policyDefaults:
+    namespace: ztp-group
+    placement:
+        labelSelector:
+            matchExpressions:
+                - key: group-du-sno
+                  operator: Exists
+    remediationAction: inform
+    severity: low
+    namespaceSelector:
+        exclude:
+            - kube-*
+        include:
+            - '*'
+    evaluationInterval:
+        compliant: 10m
+        noncompliant: 10s
+policies:
+    - name: group-du-sno-config-policy
+      policyAnnotations:
+        ran.openshift.io/ztp-deploy-wave: "10"
+      manifests:
+        - path: source-crs/PtpConfigSlave-MCP-master.yaml
+          patches:
+            - metadata:
+                name: du-ptp-slave
+              spec:
+                profile:
+                    - interface: ens5f0
+                      name: slave
+                      phc2sysOpts: -a -r -n 24
+                      ptp4lOpts: -2 -s --summary_interval -4
```

Comment:
Need to include the full CR here, not just the patch as described in the reference template. Patching lists for objects (profiles in this case, does not work yet for Custom CRs such as PTP config)